minecraft/dial.go

```diff
@@ -51,12 +52,12 @@ type Dialer struct {
 	// this field is used to obtain tokens which in turn are used to authenticate to XBOX Live.
 	// The minecraft/auth package provides an oauth2.TokenSource implementation (auth.tokenSource) to use
 	// device auth to login.
-	// If TokenSource is nil, the connection will not use authentication.
+	// If both TokenSource and Session are nil, the connection will not use authentication.
 	TokenSource oauth2.TokenSource
 
-	// XBLToken should be used in place of TokenSource if the XBL token is already known, i.e through a different
-	// oauth source. This token is for with the https://multiplayer.minecraft.net relaying party.
-	XBLToken *auth.XBLToken
+	// Session should be used in place of TokenSource if the session is already created. Using this over
+	// TokenSource will result in much faster logins, especially if you are reusing it.
+	Session *session.Session
```

Comment:
This naming and documentation is unclear. What kind of session are we talking about here? Right now, it sounds like we are talking about some sort of existing connection.